tools/PI/DevHome.PI/Models/FrameworkType.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using CommunityToolkit.Mvvm.ComponentModel;
+
+namespace DevHome.PI.Models;
+
+public partial class FrameworkType : ObservableObject
+{
+    internal bool IsExactMatch { get; private set; } = true;
+
+    internal string Determinator
+    {
+        get; set;
+    }
+
+    public string Name
+    {
+        get; set;
+    }
+
+    [ObservableProperty]
+    private bool status;
```

Comment:
What does status represent?

Reply:
FrameworkType is used to report which framework types the app uses, so status==true means it does support this type, and status==false means it does not support this type.

Reply:
Can Status be renamed to IsTypeSupported to make it more descriptive?

Reply:
Good idea. Fixed in new change.